README.rst

```diff
@@ -28,7 +38,17 @@ consecutive days.
 The "soft nurse" constraint requires that all nurses should have roughly
 even work schedules.
 
-Running the demo results in the following output:
+This demo seeks to obtain reasonable results for a nurse schedule, based on
+IKH's model. In our implementation, the number of days is D, and the number 
+of nurses is N. We wish to find a schedule for the N nurses, on the D days, 
+which satisfies the following conditions:
```

Comment:
Can be shortened, if you wish, to something like, "Our implementation attempts to find a schedule for a number N of nurses and a number D of days that satisfies the following conditions:"

Reply:
OK, thanks, @joelpasvolsky. I did it.